com.github.petebuffon.yafc.yml

```diff
@@ -0,0 +1,24 @@
+app-id: com.github.petebuffon.yafc
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: YAFC
+finish-args:
+  - "--socket=wayland"
+  - "--device=dri"
+  - "--socket=x11"
```

Comment:
does it really support wayland?

Reply:
I've tested this program with wayland and it works.

Reply:
then

```suggestion
  - "--socket=wayland"
  - "--device=dri"
  - "--socket=fallback-x11"
```

Reply:
"--socket=fallback-x11" added